sample-graphs/update.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -eu
+
+SOURCECRED_REMOTE="git@github.com:sourcecred/sourcecred.git"
+
+# Taken from https://stackoverflow.com/a/246128
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+# Use a clean SOURCECRED_DIRECTROY to avoid cache pollution.
+# Maybe reconsider this if we start storing really large projects here
+# (but putting really large projects' graphs in this Git repository is likely
+# a bad idea anyway).
+export SOURCECRED_DIRECTORY="$(mktemp -d --suffix ".sourcecred-directory")"
+
+# Just clone a fresh copy of SourceCred for convenience.
+sourcecred_repo="$(mktemp -d --suffix ".sourcecred-repo")"
+git clone "${SOURCECRED_REMOTE}" "${sourcecred_repo}"
+cd "${sourcecred_repo}"
+
+yarn && yarn backend
+
+for_repo () {
+  node bin/sourcecred.js load "$1"/"$2"
+  node bin/sourcecred.js export-graph "$1"/"$2" > "$DIR/$1_$2.json"
```

Comment:
Please consider prettifying the output (`… | jq .`) to make diffs
readable, as we do in sourcecred/sourcecred.

Reply:
Done.